bin/docker-build

```diff
@@ -30,14 +33,19 @@ if [ -n "$DOCKER_BUILDKIT_CACHE" ]; then
 fi
 
 output_params=""
+if [ -n "$DOCKER_PUSH" ]; then
+    output_params+=" --push"
+else
+    output_params+=" --load"
```

Comment:
If we're not pushing, don't we still want to load the images, for integration tests to pick them up? (in `integration_tests.yml` not in `release.yml` because in the latter case they should be pull directly from the registry).

Reply:
Hence we have this in the else condition right? 🤔 as `DOCKER_PUSH` is not enabled in `integration_tests` ?

Reply:
You're right, my bad :+1: